.xo-config.json

```
@@ -0,0 +1,25 @@
+{
+  "extends": ["xo-react", "prettier", "prettier/react"],
+  "ignore": [
+    "docs/**","spec/**"
+  ],
+
+  "overrides": [
+    {
+      "files": ["**/*.test.js", "e2e/**", "jest/*"],
+      "envs": ["browser", "jest"]
+    }
+  ],
+  "envs": ["browser"],
+  "rules": {
+    "capitalized-comments": "off",
+    "indent": "off",
+    "import/no-unassigned-import": "off",
+    "react/react-in-jsx-scope": "off",
+    "unicorn/catch-error-name": "off",
+    "unicorn/filename-case": "off",
+    "unicorn/import-index": "off",
+    "unicorn/prevent-abbreviations": "off"
+  },
+  "space": true
```

I found that I needed to amend this and specify that 4 spaces is our standard (as automated by prettier) or I got XO errors in various places.
```suggestion
  "space": 4
```

---

Happy to change it, but have not had any of those errors.